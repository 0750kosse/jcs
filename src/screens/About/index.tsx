import React from 'react';
import Profile from '../../components/Profile'
import Info from '../../components/Info'
import { profile, about } from '../../lib/data'

export default () => {

  return (
    <>
      <Profile
        header={profile.header}
        subheader={profile.subheader}
        ctaText={profile.ctaText}
        image={profile.image}
      />
      <Info
        title={about.title}
        description={about.description} />
    </>
  )
}
