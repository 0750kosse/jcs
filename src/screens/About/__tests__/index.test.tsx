import React from 'react'
import About from '../index'
import { render } from '@testing-library/react'

describe('About screen', () => {
  it('should render correclty', () => {
    const wrapper = render(<About />)
    expect(wrapper).toMatchSnapshot()
  })
})