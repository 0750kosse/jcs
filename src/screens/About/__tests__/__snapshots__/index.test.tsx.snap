// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About screen should render correclty 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-hKgILt ikMfpV"
        id="home"
      >
        <h1
          class="sc-eCssSg Tywgk profile__header"
        >
          Front End Dev
        </h1>
        <img
          class="sc-gKsewC cCEKFw"
          src="profilePic.jpeg"
        />
        <h2
          class="sc-jSgupP chcMgv"
        >
          Hey there, I'm Jose! I'm a passionate software developer. Currently living in London and originally from Barcelona. For the past two years, I've been teaching myself to code and currently seeking an entry level Front End Dev role
        </h2>
        <div
          class="sc-dlfnbm cYacxt profile__button"
        >
          <a
            aria-label="button"
            class="sc-bdfBwQ frUVuY"
            type="primary"
          >
            Contact
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz dCcfzj"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              type="primary"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            Download CV
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
        </div>
      </div>
      <div
        class="sc-iBPRYJ jIITCp"
      >
        <h1
          class="sc-fubCfw"
        >
          Projects
        </h1>
        <p
          class="sc-pFZIQ cWpMNV"
        >
          Find below a few side projects that I have been building from scratch in which I put to use the different languages learnt. Next stop in my learning bucket list are Python and Svelte, so watch this space…
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-hKgILt ikMfpV"
      id="home"
    >
      <h1
        class="sc-eCssSg Tywgk profile__header"
      >
        Front End Dev
      </h1>
      <img
        class="sc-gKsewC cCEKFw"
        src="profilePic.jpeg"
      />
      <h2
        class="sc-jSgupP chcMgv"
      >
        Hey there, I'm Jose! I'm a passionate software developer. Currently living in London and originally from Barcelona. For the past two years, I've been teaching myself to code and currently seeking an entry level Front End Dev role
      </h2>
      <div
        class="sc-dlfnbm cYacxt profile__button"
      >
        <a
          aria-label="button"
          class="sc-bdfBwQ frUVuY"
          type="primary"
        >
          Contact
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz dCcfzj"
            data-icon="external-link-alt"
            data-prefix="fas"
            focusable="false"
            role="img"
            type="primary"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
              fill="currentColor"
            />
          </svg>
        </a>
        <a
          aria-label="button"
          class="sc-bdfBwQ abOIX"
        >
          Download CV
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
            data-icon="external-link-alt"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
              fill="currentColor"
            />
          </svg>
        </a>
      </div>
    </div>
    <div
      class="sc-iBPRYJ jIITCp"
    >
      <h1
        class="sc-fubCfw"
      >
        Projects
      </h1>
      <p
        class="sc-pFZIQ cWpMNV"
      >
        Find below a few side projects that I have been building from scratch in which I put to use the different languages learnt. Next stop in my learning bucket list are Python and Svelte, so watch this space…
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
